rd-kt/rd-framework/src/commonMain/kotlin/com/jetbrains/rd/framework/AbstractBuffer.kt

```diff
@@ -64,6 +64,14 @@ abstract class AbstractBuffer {
 
     abstract fun checkAvailable(moreSize: Int)
 
+    open fun reset() {
```

Comment:
Could you please add docs to make clear the difference between these two methods